js/src/controls/Control.ts

```diff
@@ -40,14 +52,11 @@ export class LeafletControlView extends utils.LeafletWidgetView {
     var o = this.model.get('options');
     for (var i = 0; i < o.length; i++) {
       key = o[i];
-      this.listenTo(
-        this.model,
-        'change:' + key,
-        function () {
-          L.setOptions(this.obj, this.get_options());
-        },
-        this
-      );
+      this.listenTo(this.model, 'change:' + key, function () {
+        L.setOptions(this.obj, this.get_options());
+      });
```

Comment:
Using an arrow function so that the `this` context gets passed properly

```suggestion
      this.listenTo(this.model, 'change:' + key, () => {
        L.setOptions(this.obj, this.get_options());
      });
```